portfolio/src/main/webapp/script.js

```diff
@@ -12,12 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+var numDescriptor = 0;
+var timeout = setTimeout(console.log("Hello"), 1);
```

Comment:
nitpick: maybe remove debugging line? There's no need to store the return value of setTimeout.